include/internal/catch_fatal_condition.h

```diff
@@ -35,7 +35,7 @@ namespace Catch {
 
 #elif defined ( CATCH_CONFIG_POSIX_SIGNALS )
 
-#include <signal.h>
+#include <csignal>
```

Comment:
This should also remain `<signal.h>`. This file relies on POSIX-provided `<signal.h>`, rather than the C-standard `<signal.h>`.